modules/ob-creating-a-buildpacks-build.adoc

```diff
@@ -0,0 +1,265 @@
+:_mod-docs-content-type: PROCEDURE
+[id="ob-creating-a-buildpacks-build_{context}"]
+= Creating a buildpacks build
+
+Learn how to create a buildpacks build and push the created image into the target registry.
```

Comment:
For {builds-title}, the following options are available for buildpacks `ClusterBuildStrategy`:

buildpacks strategy

buildpacks-extender strategy